src/Game/Scenes/GameSceneInputHandler.cs

```diff
@@ -552,46 +552,46 @@ private bool OnLeftMouseUp()
                         }
                     }
 
-                        Mouse.LastLeftButtonClickTime = 0;
+                    Mouse.LastLeftButtonClickTime = 0;
 
-                        break;
+                    break;
 
                     case CursorTarget.SetTargetClientSide:
-                    {
```

Comment:
that intend really piss me off.... after push it's revert to new one..... and github make whole weirds comparation